beacon-chain/rpc/eth/beacon/sync_committee.go

```diff
@@ -27,6 +28,44 @@ func (bs *Server) ListSyncCommittees(ctx context.Context, req *ethpbv2.StateSync
 	ctx, span := trace.StartSpan(ctx, "beacon.ListSyncCommittees")
 	defer span.End()
 
+	currentSlot := bs.GenesisTimeFetcher.CurrentSlot()
+	currentPeriod, err := slots.SyncCommitteePeriodStartEpoch(slots.ToEpoch(currentSlot))
```

Comment:
`SyncCommitteePeriodStartEpoch` returns epoch not period, variable name `currentPeriod` is probably not correct

Reply:
Yeah, it's just the variable name, changing it now and also fixing the deepsource complain